doc/nrf/ug_nrf52_developing.rst

```diff
@@ -90,6 +88,30 @@ Consider using these features in your project to speed up the FOTA upgrade proce
 
 .. fota_upgrades_end
 
+.. ug_nrf52_developing_fota_in_mesh:
+.. fota_upgrades_bt_mesh_start
+
+FOTA in Bluetooth mesh
+======================
+
+In |NCS|, there is currently no support for Device Firmware Update (DFU) over Bluetooth mesh when performing FOTA.
+However, support for point-to-point DFU over Bluetooth Low Energy using MCUmgr component is available.
+
+To perform FOTA upgrade when working with the Bluetooth mesh protocol, use DFU over Bluetooth LE on either smartphone or PC command line tool, following the instructions described in `FOTA over Bluetooth Low Energy`_ above.
```

Comment:
Bluetooth LE or Bluetooth Low Energy, mix in these files.
Is the "use DFU over Bluetooth LE on either smartphone or PC command line tool" necessary? Is that not described in 'FOTA over Bluetooth Low Energy'